Analysis/DataModel/include/Analysis/ReducedInfoTables.h

```diff
@@ -34,7 +35,7 @@ DECLARE_SOA_TABLE(ReducedEvents, "AOD", "REDUCEDEVENT", o2::soa::Index<>,
                   collision::PosX, collision::PosY, collision::PosZ, collision::NumContrib);
 
 DECLARE_SOA_TABLE(ReducedEventsExtended, "AOD", "REEXTENDED",
```

Comment:
Should we add DQ to these names? Or is there a reason against?

Reply:
Hi Jan,
I can add DQ in the names, sure.
Ionut

Reply:
Hi Jan, thinking more on the naming issue,
i think i would prefer adding all this in a "DQ" namespace.
In this way i can use shorter names and i do not impose on how other groups will 
do their naming in the future.
If its fine with you, maybe we discuss a bit on this during the group meeting and i make a separate PR for naming, depending on what comes up.
Ionut